pep-0675.rst

```diff
@@ -868,6 +881,40 @@ the ``Template`` API to only accept ``Literal[str]``:
         def __init__(self, source: Literal[str]): ...
 
 
+Logging Format String Injection
+-------------------------------
+
+Logging frameworks often allow their input strings to contain
+formatting directives. At its worst, allowing users to control the
+logged string has led to
+`CVE-2021-44228<https://nvd.nist.gov/vuln/detail/CVE-2021-44228>`_
```

Comment:
```suggestion
`CVE-2021-44228 <https://nvd.nist.gov/vuln/detail/CVE-2021-44228>`_
```

A space is required between the inline text and the URL target. This is causing the build to fail.